internal/buildscripts/packaging/technical-addon/Splunk_TA_otel/README.md

```diff
@@ -0,0 +1,144 @@
+# Splunk Add-on for OpenTelemetry Collector
+
+This technical add-on for Splunk Universal Forwarder helps in deploying the OpenTelemetry Collector alongside Universal Forwarder. 
+
+Splunk Add-on for OpenTelemetry Collector is supported on Linux and Windows on `amd64` (`x86_64`)
+
+## Contents
+
+The package contains the following folders and files -
+
+------------|
+                 |------windows_x86_64
+
+------------|
+                 |------linux_x86_64
+                 
+------------|
+                 |------README
+                 
+------------|
+                 |------default
+                 
+------------|
+                 |------configs
+            
+------------|
+                 |------README.md
+
```

Comment:
nit, feel free to ignore: this should be in a code block to look like a folder structure